test/remote/gateways/remote_pin_test.rb

```diff
@@ -19,10 +19,9 @@ def setup
 
     @additional_options_3ds = @options.merge(
       three_d_secure: {
-        version: '1.0.2',
-        eci: '06',
-        cavv: 'AgAAAAAAAIR8CQrXcIhbQAAAAAA',
-        xid: 'MDAwMDAwMDAwMDAwMDAwMzIyNzY='
+        "enabled": true,
+        "fallback_ok": true,
+        "callback_url": "https://yoursite.com/authentication_complete"
```

Comment:
Would be good to retain the old test here and also add the new version, perhaps updating to `@additional_options_3ds_passthrough = {version, eci, etc...}` and `@additional_options_3ds = {enabled, falback_ok, etc...}`, updating `test_successful_authorize_and_capture_with_passthrough_3ds` to use `@additional_options_3ds_passthrough` and adding `test_successful_authorize_and_capture_with_3ds`

Reply:
I've updated the tests.